components/empty-state/test/empty-state-text.visual-diff.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <title>d2l-empty-state-text</title>
+    <script type="module">
+        import '../..//typography/typography.js';
+        import '../empty-state-text.js';
+        import '../../button/button-subtle.js';
+        import '../../link/link.js';
+    </script>
+</head>
+<body class="d2l-typography">
+
+    <div class="visual-diff">
```

Comment:
Need to add margins to separate different visual diff tests

Reply:
Most other visual-diff test files include a reference to :

``` html
<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
```

It include some useful styles you'll probably want.
https://github.com/BrightspaceUI/core/blob/main/test/styles.css

Reply:
Thanks!